src/Tribe/RSVP.php

```diff
@@ -1359,7 +1365,11 @@ public function send_tickets_email( $order_id, $event_id = null ) {
 		 */
 		$content = apply_filters( 'tribe_rsvp_email_content', $this->generate_tickets_email_content( $to_send ), $event_id, $order_id );
 
-		wp_mail( $to, $subject, $content, $headers, $attachments );
+		$sent = wp_mail( $to, $subject, $content, $headers, $attachments );
```

Comment:
:warning: `wp_mail` should be used sparingly. For any bulk emailing should be handled by a 3rd party service, in order to prevent domain or IP addresses being flagged as spam.

<sub>hash: 2e3ac060611382ed663eb976977d4f15</sub>